src/main/java/net/minecraftforge/fluids/FluidAttributes.java

```diff
@@ -229,20 +300,27 @@ public Component getDisplayName(FluidStack stack)
 
     /**
      * A FluidStack sensitive version of getTranslationKey
+     * @param stack the provided {@code FluidStack} to get the display name of.
+     * @return Returns the translation key of the provided {@code FluidStack} as a string.
      */
     public String getTranslationKey(FluidStack stack)
     {
         return this.getTranslationKey();
     }
 
     /**
-     * Returns the translation key of this fluid.
+     * @return Returns the translation key of this fluid.
      */
     public String getTranslationKey()
```

Comment:
I feel like this method potentially should be deprecated/slated to be removed for 1.18 as I can't see much reason for it given there is a more sensitive version and if a mod overrides one but not the other then things may behave unexpectedly.